docs/excel/configure-your-add-in-to-use-a-shared-runtime.md

```diff
@@ -88,7 +88,24 @@ Follow these steps for a new or existing project to configure it to use a shared
    ...
    ```
 
-8. Save your changes and rebuild the project.
+8. Make sure the taskpane.html has a `<script>` tag that references the dist/functions.js file. The following is an example:
+
+```html
+<script type=”text/javascript” src=”/dist/functions.js” ></script>
+```
```

Comment:
This should address the build suggestion
```suggestion
    ```html
    <script type=”text/javascript” src=”/dist/functions.js” ></script>
    ```
```

Reply:
fixed